worker/task.go

```diff
@@ -235,7 +235,7 @@ func processTask(q *task.Query, gid uint32) (*task.Result, error) {
 		for _, uid := range uids.Uids {
 			key := x.DataKey(attr, uid)
 			pl, decr := posting.GetOrCreate(key, gid)
-
+			defer decr()
```

Comment:
only potential flaw in this that now posting's reference count will decrease only at function unwinding,
not immediately in each iteration of for loop.
Can that be a concern ? What if uids.Uids are larger ?